cmd/ps.go

```diff
@@ -56,7 +56,9 @@ func PsScale(appID string, targets []string) error {
 			fmt.Printf("'%s' does not match the pattern 'type=num', ex: web=2\n", target)
```

Comment:
i am ok with returning and error but i think we are not returning an error because user can try to scale multiple proc types at a time and only few can be bad.